specification/app/resource-manager/readme.md

```diff
@@ -66,6 +66,11 @@ input-file:
   - Microsoft.App/preview/2024-10-02-preview/SourceControls.json
   - Microsoft.App/preview/2024-10-02-preview/Subscriptions.json
   - Microsoft.App/preview/2024-10-02-preview/Usages.json
+directive:
+  - suppress: PatchBodyParametersSchema
+    from: SessionPools.json
+    reason: |
+      Session Pool is using componentType as the discriminator. While the discriminator is a required property, this rule prevent it being present in the patch request body.
```

Comment:
Why does session pool not support patch?

Reply:
It do support patch, but during patch(common type) identity is not required and in common type it marked as required. This is suppressed suggested by ARM reviewer above [rkmanda](https://github.com/rkmanda) "For the patch... rule that is breaking please add a suppression.."

Reply:
Thank you for the explanation